spec/controllers/supplejack_api/concerns/user_sets_controller_metrics_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+
+describe ApplicationController, type: :controller do
+controller do
+  include SupplejackApi::Concerns::UserSetsControllerMetrics
+
```

Comment:
I haven't seen this type of test before, where you make a controller.  Is the indentation meant to be like that?  It's the same on al the others.

Reply:
Oh yeah the controller should be indented in one. It's a way of creating an anonymous controller so I can test that the concern works as intended when it is included. 

https://relishapp.com/rspec/rspec-rails/docs/controller-specs/anonymous-controller